raspberry-pi/5/overlay.nix

```diff
@@ -0,0 +1,6 @@
+final: _prev: {
+  linux_rpi5 = final.linux_rpi4.override {
+    rpiVersion = 5;
+    argsOverride.defconfig = "bcm2712_defconfig";
```

Comment:
Out of curiosity. Where is such a config defined? I believe it's straight from the upstream Linux kernel, but would you have a url where we can track rpi5 support there?

Reply:
It's in the downstream https://github.com/raspberrypi/linux kernel. The value for `linux_rpi4` is `bcm2711_defconfig`.